generators/app/templates/src/doc.json

```diff
@@ -205,16 +194,15 @@
         "examples": {
           "application/json": {
             "errors": [
-              {
-                "message": "Missing field",
-                "field": "email"
-              }
+              { "message": "User not found" },
+              { "message": "CRM not found"},
+              { "message": "And so on"}
             ]
           }
         }
       },
-      "NotFound": {
-        "description": "Not Found. The server has not found anything matching the identifier.",
+      "InternalServerError": {
+        "description": "Internal Server Error. An unexpected condition prevented the service from fulfilling the request.",
         "schema": {
           "type": "object",
           "properties": {
```

Comment:
@brunogsa what do you think about extracting this to a "#/definition/Errors" description?